### PR TITLE
Storage station requires free input and output sides

### PR DESCRIPTION
**Pull request by thomerz**
Similar to other MPS types (besides DS), SS should block zones at input and output side, respectively. DS block their input side only since no product can be fetched from a DS.

### CODE REVIEW

**Comment by morxa** [comment]
This needs to be rebased due to changes merged from #16. Do you want me to do that?

File: rulebook.tex [hunk 1]
```diff
@@ -709,13 +709,13 @@ \subsubsection{Machine Positioning}
 
 \begin{figure}[t!]
     \centering
-    \subfigure[For rotations with \ang{0}, \ang{90}, \ang{180} or \ang{270}. BS, CS, and RS block the zones close to the input as well as the output. DS and SS block the zones close to the input respectively the output.]{
+    \subfigure[For rotations with \ang{0}, \ang{90}, \ang{180} or \ang{270}. BS, CS, RS, and SS block the zones close to the input as well as the output. DS block the zones close to the input.]{
```
Comment (timn) [inline rulebook.tex]:
```suggestion
    \subfigure[For rotations with \ang{0}, \ang{90}, \ang{180} or \ang{270}. BS, CS, RS, and SS block the zones close to the input as well as the output. DS blocks the zones close to the input.]{
```
Missing singular s in verb.

File: rulebook.tex [hunk 2]
```diff
@@ -709,13 +709,13 @@ \subsubsection{Machine Positioning}
...
         \label{fig:zone-mps-rotation-straight}%
         \centering
         \includegraphics[width=\textwidth]{bZones1.pdf}
     }
     \quad
-    \subfigure[For rotations with  \ang{45}, \ang{135}, \ang{225} and \ang{315}. BS, CS, and RS block the zones close to the input as well as the output. DS and SS block the zones close to the input respectively the output.]{
+    \subfigure[For rotations with  \ang{45}, \ang{135}, \ang{225} and \ang{315}. BS, CS, RS, and SS block the zones close to the input as well as the output. DS block the zones close to the input.]{
```
Comment (timn) [inline rulebook.tex]:
```suggestion
    \subfigure[For rotations with  \ang{45}, \ang{135}, \ang{225} and \ang{315}. BS, CS, RS, and SS block the zones close to the input as well as the output. DS blocks the zones close to the input.]{
```
Same here.

File: rulebook.tex (hunk 1 above)
Comment (timn) [reply rulebook.tex]: And shorten the lines to 80 characters.

**Comment by morxa** [comment]
> This needs to be rebased due to changes merged from #16. Do you want me to do that?

I rebased onto the latest master.

File: rulebook.tex (hunk 1 above)
Comment (morxa) [reply rulebook.tex]: I fixed this to get this change going

**Comment by morxa** [comment]
+1

**Comment by MostafaGomaa** [comment]
+1

**Comment by dka14** [comment]
+1

**Comment by morxa** [comment]
__Proposal accepted__
* in favor: 2
* abstentions: 0
* opposed: 0

**Pull request merged**